addons/common/XEH_preInit.sqf

```diff
@@ -18,8 +18,6 @@ SCRIPT(XEH_preInit);
  * and will hopefully ensure forward compatibility with future ArmA II patches.
  */
 
-if (isNil "CBA_FUNC_RECOMPILE") then { CBA_FUNC_RECOMPILE = CACHE_DIS(functions) };

```

Comment:
Used in https://github.com/CBATeam/CBA_A3/blob/master/addons/common/init_functionsModule.sqf#L13 (now nil)